BWT/BWT.cs

```diff
@@ -0,0 +1,104 @@
+public static class BWT
+{
+    public static (string, int) Transform(string str)
+    {
+        var BWTString = new System.Text.StringBuilder();
+        var sortedShifts = SuffixArrayBuild(str);
+        var originalStringIndex = 1;
+        for (var i = 0; i < str.Length; ++i)
+        {
+            if (sortedShifts[i] == 0)
+            {
+                originalStringIndex = i + 1;
+                BWTString.Append(str[^1]);
+            }
+            else
+            {
+                BWTString.Append(str[sortedShifts[i] - 1]);
+            }
+        }
+        return (BWTString.ToString(), originalStringIndex);
```

Comment:
Поставьте [StyleCop](https://github.com/StyleCop/StyleCop)

```suggestion
        }
        
        return (BWTString.ToString(), originalStringIndex);
```